db/revert/tables/ccbc_user_001_no_unique_email.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ccbc:tables/ccbc_user_001_no_unique_email from pg
+
+begin;
+
+create unique index ccbc_user_email_address on ccbc_public.ccbc_user(email_address);
```

Comment:
Wouldn't this, on revert, fail if someone has two accounts with the same email?

Reply:
Possibly, but reverts are for local development only